packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_control_pane.dart

```diff
@@ -15,24 +15,41 @@ class SnapshotControlPane extends StatelessWidget {
       : super(key: key);
 
   final DiffPaneController controller;
+  static const _classFilterWidth = 200.0;
 
   @override
   Widget build(BuildContext context) {
     return ValueListenableBuilder<bool>(
       valueListenable: controller.isProcessing,
       builder: (_, isProcessing, __) {
-        final current = controller.selectedItem as SnapshotListItem;
+        final current = controller.core.selectedItem as SnapshotInstanceItem;
 
         return Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
           children: [
-            const SizedBox(width: defaultSpacing),
-            if (!isProcessing && current.heap != null) ...[
-              _DiffDropdown(
-                current: current,
-                list: controller.snapshots,
-              ),
-              const SizedBox(width: defaultSpacing),
-            ],
+            Row(
+              children: [
+                const SizedBox(width: defaultSpacing),
```

Comment:
do we really want a SizedBox in the beginning of the list? let's remove

Reply:
We need to add space before 'Diff':
<img width="370" alt="Screen Shot 2022-10-05 at 12 52 16 PM" src="https://user-images.githubusercontent.com/12115586/194150310-7c9ef388-9704-4ea8-9996-1b1cf01f00a2.png">

Reply:
We should wrap the outer row in a Padding widget to make the intent clear. I suspect we'll want the padding on both sides of this Row, so we can use `EdgeInsets.symmetric`